test/unit/modules/tableEmbed.spec.ts

```diff
@@ -47,15 +47,15 @@ describe('tableHandler', () => {
           {
             insert: {
               'table-embed': {
-                rows: [
+                rows: new Delta([
```

Comment:
Tests should not be changed unless it's related to types.

Reply:
See comment here: https://github.com/quilljs/quill/pull/3845#discussion_r1287726794